lib/db/mixin_database.dart

```diff
@@ -270,18 +270,16 @@ class MixinDatabase extends _$MixinDatabase {
           .isNotEmpty;
 }
 
-QueryExecutor _openConnection(File dbFile) {
-  final vmDatabase = NativeDatabase(dbFile);
-  if (!kDebugMode) {
-    return vmDatabase;
-  }
-  return CustomVmDatabaseWrapper(vmDatabase, logStatements: true);
-}
+QueryExecutor _openConnection(File dbFile) => CustomVmDatabaseWrapper(
+      NativeDatabase(dbFile),
+      logStatements: true,
+      explain: kDebugMode,
+    );
 
 /// Connect to the database.
 Future<MixinDatabase> connectToDatabase(
   String identityNumber, {
-  bool fromMainIsolate = false,
```

Comment:
no need to change this. 

if `connectToDatabase(fromMainIsolate: true)` called from other isolate, it will cause unexpected behaivor.